_posts/2019-07-30-microfrontends-holidaycheck-part2.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title:  "Micro Frontends at HolidayCheck (part 2) - Speed"
+categories: microfrontend javascript 
+author_name: Sebastian Bruckner
+author_url : /author/sebastianbruckner
+author_avatar: sebruck
+read_time : 10
+feature_image: posts/2019-06-08-browsertools-2/ben-harritt-75854-unsplash.jpg
+---
+
+In the [part 1](/post/2019/06/28/microfrontends-holidaycheck-part1) of this series, I describe why and how we do micro frontends at HolidayCheck. One of the key
+aspects I described was the focus on speed. Our first micro frontend, the header, may not slow down the pages
```

Comment:
I think it would be more understandable like:
`Our goal was that our first micro frontend, the header, doesn't slow down the pages on which it is embedded.`